models/SingleUserAttendance.ts

```diff
@@ -0,0 +1,18 @@
+import * as t from "io-ts";
```

Comment:
can you delete this file and make sure all *.ts files in models are ignored.

Reply:
won't deleting the model create errors?